opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/dsl/MetricAggregationBuilderTest.java

```diff
@@ -5,6 +5,7 @@
 
 package org.opensearch.sql.opensearch.storage.script.aggregation.dsl;
 
+import static java.util.Collections.singletonList;
```

Comment:
I would recommend changing everything to List.of and remove this dependency.

Reply:
Done. Replaced all occurrences of `Collections.singletonList` with `List.of`.